tests/e2e/utils/constants.py

```diff
@@ -4,6 +4,7 @@
 
 DEFAULT_HOST = "http://localhost:8080/"
 DEFAULT_USER_NAMESPACE = "kubeflow-user-example-com"
+DEFAULT_NAMESPACE = "kube-system"
```

Comment:
Would it make sense to rename this to something like `DEFAULT_SYSTEM_NAMESPACE` to avoid confusion with the `default` namespace?

Reply:
done.